# perception config (todo give each node separate cfg)

# track_iface
dt_track_iface: 1.0
N_delay_pub_pathglobal: 1
N_delay_repub_pathglobal: 5

# state_est_cart 
dt_state_est_cart: 0.02

# state_est_frenet
dt_state_est_frenet: 0.02

# road percept
dt_road_perception: 0.25
N_pathlocal: 30
ds_pathlocal: 0.25
s_rel_start_pathlocal: 1 # distance behind vehicle where local path starts


